### PR TITLE
[CFM_47] Script to setup dev database environment

### PR DESCRIPTION
**Pull request by 0916dhkim**
[Trello ticket](https://trello.com/c/54YjlIKW)

#### Changes
- Add `docker-compose.yml` file for local DynamoDB & cfm-network.
- Add `schema/*.json` files for database schema.
- Add `scripts/create_local_dynamodb_tables.py`
- Update README accordingly.

#### Testing
1. Start a local DynamoDB with `docker compose up`
2. Create tables by running `./scripts/create_local_dynamodb_tables.py`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,22 +25,17 @@ Confirm that the following requests work for you
...
-    3. **fridge_history:** `aws dynamodb create-table --table-name fridge_history --attribute-definitions AttributeName=fridge_id,AttributeType=S AttributeName=timestamp,AttributeType=N --key-schema AttributeName=fridge_id,KeyType=HASH AttributeName=timestamp,KeyType=RANGE --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5 --endpoint-url http://localhost:8000`
-    4. **tag**: `aws dynamodb create-table --table-name tag --attribute-definitions AttributeName=tag_name,AttributeType=S --key-schema AttributeName=tag_name,KeyType=HASH  --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5 --endpoint-url http://localhost:8000`
-4. **Build the functions inside a Docker container**
-    1. `sam build --use-container`
-5. **Load data into your local Dynamodb tables**
+1. **Start a local DynamoDB service**
+    ```sh
+    docker compose up
```
Comment (CollectiveFocus) [inline README.md:30]: Could you add `docker-compose up -d` too for running this in the background?

File: README.md [hunk 2]
```diff
@@ -25,22 +25,17 @@ Confirm that the following requests work for you
...
+    ```
+1. **Create tables**
+    ```sh
+    ./scripts/create_local_dynamodb_tables.sh
```
Comment (CollectiveFocus) [inline README.md:34]: Getting the following error when running this command:

```
./scripts/create_local_dynamodb_tables.sh: line 2: realpath: command not found

Error parsing parameter '--cli-input-json': Unable to load paramfile file:///*.json: [Errno 2] No such file or directory: '/*.json'
Failed to create table from /*.json
```

File: README.md (hunk 1 above)
Comment (0916dhkim) [reply README.md:30]: That's a good point. Applied in 78a2baa7d961e9e0be71f2f5956c6fc82640113c

File: README.md (hunk 2 above)
Comment (0916dhkim) [reply README.md]: I didn't know `realpath` was not available in some OS. Fixed in d5b9829dc453d2088ac8770e76ba0ac96970e4d2.
It should work when you run ./scripts/create_local_dynamodb_tables.**py**

File: README.md [hunk 3]
```diff
@@ -25,22 +25,19 @@ Confirm that the following requests work for you
...
+    ```sh
+    docker compose up
+    # OR if you want to run it in the background:
+    docker compose up -d
+    ```
+1. **Create tables**
+    ```sh
+    ./scripts/create_local_dynamodb_tables.py
+    ```
+1. **Load data into your local Dynamodb tables**
```
Comment (1jeanpaul1) [inline README.md:38]: All of the commands below are ran from within `CommunityFridgeMapApi/`

Would be good to add a `$CommunityFridgeMapApi` before each command or a step to `cd CommunityFridgeMapAp/i`

**Comment by 1jeanpaul1** [comment]
I'm so excited! Looks and works great :)

Our Copy & Paste frustration will soon come to an end

Just made one comment but other than that I tested locally and it works perfect :)

File: README.md (hunk 3 above)
Comment (0916dhkim) [reply README.md:39]: Good catch :+1: 
Fixed in 47ac0ac2e89e6673d3c91fbdddc24f9ca832d222

**Review by 1jeanpaul1 — APPROVED** [review]
Looks good to me!

**Pull request merged**